### PR TITLE
NTR dermal chromatophore

### LINKED ISSUE #2307: [NTR] dermal chromatophore
**Preferred term label** dermal chromatophore

**Definition** A chromatophore that is part of the dermis. PMID:28553007

**Parent cell type term** pigment cell CL:0000147

**Anatomical structure where the cell type is found** dermis UBERON:0002067

**Your ORCID** 0000-0002-0819-0473

**Additional note**
Mammals and birds have melanocytes as the sole type of pigment cells. Reptiles, amphibians, and fish possess a variety of pigment cells collectively termed chromatophores. Chromatophores in reptiles are typically localized in the dermis, which contrasts with mammalian epidermal melanocytes. Few reptile species are reported to also possess epidermal melanocytes. Chromatophores or chromatophore-like cells (not melanomacrophages) are also commonly encountered in the coelom, especially in the parietal coelomic membrane of lizards, and scattered through the lung and liver. PMID:35324843

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2307

### CODE REVIEW

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Thank you for submitting the PR.
Please, consider the suggested changes.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -31920,6 +31921,15 @@ AnnotationAssertion(rdfs:label obo:CL_4040005 "mesenchymal stem cell of apical p
 SubClassOf(obo:CL_4040005 obo:CL_0000134)
 SubClassOf(obo:CL_4040005 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003677))
 
+# Class: obo:CL_4040006 (dermal chromatophore)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_4040006 "A chromatophore that is part of the dermis.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040006 <https://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040006 "2024-03-07T09:37:14Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:28553007") rdfs:label obo:CL_4040006 "dermal chromatophore")
+SubClassOf(obo:CL_4040006 obo:CL_0000147)
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:31930]: I think it is good as it is.  A logical def with these axioms would be much too broad.

**Comment by paulacalipho** [comment]
@aleixpuigb @dosumis Thank you for reviewing. I get this warning, and I do not understand what it means and what I should do
image

**Comment by anitacaron** [comment]
@paulacalipho, no worries. It's just a difference between Operational Systems and how they interpret the line ending in the file.

**Comment by paulacalipho** [comment]
@aleixpuigb I understood I did not need to do the logical definition, right? Is there anything else to improve?

**Review by aleixpuigb — APPROVED** [review]
>@aleixpuigb I understood I did not need to do the logical definition, right? Is there anything else to improve?

That's correct, the logical definition is not needed. Everything looks good, thank you!

**Comment by paulacalipho** [comment]
@aleixpuigb thanks, It is indicated merging is blocked. Could you do it for me?

**Pull request merged**